tests/rbd/test_rbd_disallow_group_snapshot_rollback.py

```diff
@@ -0,0 +1,387 @@
+            if rc:
+                log.error(f"Image {ispec} add to group {group_spec} failed.")
+                return 1
+            else:
+                log.info(f"Image {ispec} add to group {group_spec} succeeded.")
+
+        # Create group snapshot
+        snap = kw.get("config", {}).get("snap", "group_snap_default")
+        rc = create_snap_and_verify(
+            client=client, pool=test_pool, group=test_group, snap=snap
+        )
+        if rc:
+            log.error("Group snap creation failed")
+            return 1
+        else:
+            log.info("Group snap creation with validation done")
+
+        # Create a new RBD image (image3) and add it to the group.
+        new_pool = "new_pool_" + random_string()
+        log.info(f"Create a new pool {new_pool}")
+        config = kw.get("config", {})
+        is_ec_pool = True if "ec" in pool_type else False
+        new_image = "image_" + random_string(len=5)
+        new_pool_config = {new_image: {"size": "1G", "io_total": "1G"}}
+        image_spec.append(f"{new_pool}/{new_image}")
+        if is_ec_pool:
+            data_pool_new = "data_pool_new_" + random_string()
+            new_pool_config["data_pool"] = data_pool_new
+        rc = create_single_pool_and_images(
+            config=config,
+            pool=new_pool,
+            pool_config=new_pool_config,
+            client=client,
+            cluster="ceph",
+            rbd=rbd,
+            ceph_version=int(config.get("rhbuild")[0]),
+            is_ec_pool=is_ec_pool,
+            is_secondary=False,
+        )
+        if rc:
+            log.error(f"Creation of pool {new_pool} failed")
+            return rc
+        # Adding the new pool details to config so that they are handled in cleanup
+        if pool_type == "rep_pool_config":
+            kw["config"]["rep_pool_config"][new_pool] = {}
+        elif pool_type == "ec_pool_config":
+            kw["config"]["ec_pool_config"][new_pool] = {"data_pool": data_pool_new}
+
+        add_image_group_kw = {
+            "group-spec": group_spec,
+            "image-spec": image_spec[2],
+            "client": client,
+        }
+        rc = add_image_to_group_and_verify(**add_image_group_kw)
+        if rc:
+            log.error(f"Image {new_image} add failed, test case fail")
+            return 1
+        else:
+            log.info(f"Image {new_image} add succeeded")
+
+        for j in range(0, len(image_spec)):
+            io_config["config"]["image_spec"] = [image_spec[j]]
+            krbd_io_handler(**io_config)
+
+        md5_after_snap = []
+        for i in range(0, len(image_spec)):
+            md5 = get_md5sum_rbd_image(
+                image_spec=image_spec[i],
+                rbd=rbd,
+                client=client,
+                file_path="file" + random_string(len=5),
+            )
+            md5_after_snap.append(md5)
+        log.info(f"md5 sums after snap: {md5_after_snap}")
```

Comment:
please keep it it is just `sum`

Reply:
md5_after_snap is a list of sums of 3 images. Hence sums.